media_driver/agnostic/common/codec/hal/codechal_decode_scalability.cpp

```diff
@@ -1499,7 +1499,10 @@ MOS_STATUS CodecHalDecodeScalability_SetHintParams(
     else
     {
         VEParams.bScalableMode = false;
-        CODECHAL_DECODE_CHK_STATUS_RETURN(pVEInterface->pfnVESetHintParams(pVEInterface, &VEParams));
+        if (pVEInterface->pfnVESetHintParams)
```

Comment:
This looks to be a patch which can be merged in right now.

Reply:
Yes, I will submit a PR to check-in this patch first.

Reply:
Done with PR#481 https://github.com/intel/media-driver/pull/481